source/_integrations/homeassistant_sky_connect.markdown

```diff
@@ -12,13 +12,43 @@ ha_integration_type: hardware
 ha_config_flow: true
 ha_platforms:
   - update
+related:
+  - url: https://support.nabucasa.com/hc/en-us/categories/24734620813469-Home-Assistant-Connect-ZBT-1
+    title: Home Assistant Connect ZBT-1 documentation
+  - docs: /connectzbt1/
+    title: Home Assistant Connect ZBT-1 product page
+
 ---
 
 The Home Assistant Connect ZBT-1 integration provides hardware information for the hardware configuration page.
 
-For documentation on the Home Assistant Connect ZBT-1, please visit the [documentation page](https://connectzbt1.home-assistant.io/documentation/).
-If you are looking to buy one, please visit the [product page](https://home-assistant.io/connectzbt1)
+For documentation on the Home Assistant Connect ZBT-1, please visit the [documentation page](https://support.nabucasa.com/hc/en-us/categories/24734620813469-Home-Assistant-Connect-ZBT-1).
+If you are looking to buy one, please visit the [product page](/connectzbt1)
 
 ## Configuration
 
 This integration is not user configurable.
+
+## Re-installing the firmware using the Silicon Labs Web Flasher
+
+The Home Assistant Connect ZBT-1 ships with a full-fledged Zigbee firmware. At this point, there are no known issues with the factory-flashed firmware.
+
+Normally, there is no need to re-install the firmware. Firmware updates for the Connect ZBT-1 are natively supported in Home Assistant 2025.4 and newer if you are running the ZHA integration or OTBR.
+
+If your Home Assistant installation has no add-ons available, you can use the web flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB.
```

Comment:
Firmware updates don't rely on addons so maybe we can adjust the wording to be explicit about the situations where you'd need to use the web flasher?

```suggestion
If you are not using ZHA or OTBR, you can use the web flasher below to install the latest firmware version for Home Assistant Connect&nbsp;ZBT-1 directly from your browser over USB.
```